xbmc/windows/GUIMediaWindow.cpp

```diff
@@ -727,30 +713,36 @@ bool CGUIMediaWindow::GetDirectory(const std::string &strDirectory, CFileItemLis
 // \brief Set window to a specific directory
 // \param strDirectory The directory to be displayed in list/thumb control
 // This function calls OnPrepareFileItems()
-bool CGUIMediaWindow::Update(const std::string &strDirectory, bool updateFilterPath /* = true */)
+bool CGUIMediaWindow::Update(const std::string &_arg, bool updateFilterPath /* = true */)
```

Comment:
IMO we should leave it as it is, because each sub class use the same parameter name. Adjust the method doc and make it clear there. We could think about a better naming for it and do it the right way.